### DISCUSSION TITLE
Lima and docker-compose support

### DISCUSSION THREAD

**Discussion by restagner** [lima-vm/lima#469] [Q&A] [answered]
Currently, I'm using Docker Desktop for Mac to support a project at work.  I rely heavily on using docker-compose to start/stop a set of services required to test an application. 

Does Lima support reading from a docker-compose.yml file and launching the services contained within this file?

**Comment by AkihiroSuda** [comment, marked as answer]
Yes, `lima nerdctl compose up` should work.

You can also use Docker with Lima, if nerdctl doesn’t work for you. https://github.com/lima-vm/lima/blob/v0.7.4/examples/docker.yaml

**Reply by restagner** [reply]
Thanks

**Reply by delmicio** [reply]
does it support the `build` parameter? it is not working on my M1 pro 
```
lima nerdctl compose build lms 
FATA[0000] arguments [lms] not supported      
```

**Reply by dbulaj** [reply]
@delmicio did you manage to build with lima's `compose` ?

**Reply by delmicio** [reply]
> @delmicio did you manage to build with lima's `compose` ?

Sorry I don't remember. I abandoned my attempt to use Lima.

**Comment by jhult** [comment]
For [Podman](https://github.com/lima-vm/lima/blob/v0.8.0/examples/podman.yaml), you can use this command: `lima podman-compose up`
